### ISSUE TITLE
Crashes (sometimes) on middle-mouse pasting

### ISSUE THREAD

**Issue by blueyed** [raelgc/scudcloud#416]
I sometimes see crashes after pasting the mouse selection on X11 / Arch Linux.

```
% scudcloud --version
/usr/lib/python3.5/site-packages/pkg_resources/__init__.py:549: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  return get_distribution(dist).load_entry_point(group, name)
ScudCloud 1.18
Python 3.5.1
Qt 4.8.7
PyQt 4.11.4
SIP 4.17
```

It's not reproducible really, but seems to be triggered by something in the content.

There's no entry in the list of core dumps via `coredumpctl`.

**Comment by raelgc** [comment]
Hi @blueyed and thanks for reporting this issue.

We should be able to reproduce this in a consistent way. Is usually just text in the mouse content?

**Comment by blueyed** [comment]
Yep, but it might have some formatting applied?! - either that or something like unclosed curly braces, too much bytes or too many (potentially unbalanced) newlines...

**Comment by blueyed** [comment]
I am using CopyQ for clipboard management, which might have an influence.
